packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -293,32 +293,36 @@ async function setupServices(
 
   const governorCreatorFacet = consume.vaultFactoryGovernorCreator;
   /** @type {Promise<VaultFactory & LimitedCreatorFacet<any>>} */
-  const vaultFactoryCreatorFacet = /** @type { any } */ (
+  const vaultFactoryCreatorFacetP = /** @type { any } */ (
     E(governorCreatorFacet).getCreatorFacet()
   );
 
   // Add a vault that will lend on aeth collateral
-  const aethVaultManagerP = E(vaultFactoryCreatorFacet).addVaultType(
+  /** @type {Promise<VaultManager>} */
+  const aethVaultManagerP = E(vaultFactoryCreatorFacetP).addVaultType(
     aethIssuer,
     'AEth',
     rates,
   );
-  /** @type {[any, VaultFactory, VFC['publicFacet']]} */
-  // @ts-expect-error cast
   const [
     governorInstance,
-    vaultFactory,
+    vaultFactoryCreator,
```

Comment:
this renaming causes a lot of churn. If you really want it, the uses below need to be updated as well so we dont' have a bunch of `vaultFactory: vaultFactoryCreator`. Given that the type system clarifies that it's the creator facet, this seems unnecessary.